scripts/generate_logit_filters.sh

```diff
@@ -1,9 +1,45 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-set -euo pipefail
+# Pull together the various components of the logstash filters from their various sources
+# and combine them into a single file for consumption by logit.
 
-echo "filter {"
-sed 's/^/  /' < config/logit/10_base.conf
-sed 's/^/  /' < config/logit/20_logsearch_for_cloudfoundry.conf
-sed 's/^/  /' < config/logit/30_custom_filters.conf
-echo "}"
+set -eo pipefail
+
+logsearch_boshrelease_tag=$1
+
+if [ -z "${logsearch_boshrelease_tag}" ]
+then
+    echo "logsearch boshrelease tag not set. Is LOGSEARCH_BOSHRELEASE_TAG set?"
+    exit 1
+fi
+
+set -u
+
+apk update && apk add git
+
+cd /tmp
+git clone https://github.com/cloudfoundry-community/logsearch-for-cloudfoundry.git
+cd logsearch-for-cloudfoundry/src/logsearch-config
```

Comment:
I think we should pin the version of logsearch-for-cloudfoundry we're using as well. This ensures we get a consistent output.

Reply:
The files we reference are via a tag passed in as an argument...I'm not sure I follow here. Are you suggesting that instead of allowing someone to specify the version when we run it they have to update the script every time they want it to change? I was trying to avoid having to remember to update a version in a script to run it. Could you elaborate?

Reply:
I was pointing out that we're consuming 2 different git repos in this script, but only pinning a version for one of them - the script will always use current master of `logsearch-for-cloudfoundry`.

As for pinning versions in the repo, I think it's fine to have this script parameterised, but I think we need to pin the versions somewhere in code (maybe in the `Makefile`) so that it's clear exactly what was used to generate the current filters.